### ISSUE TITLE
how to input the dataset 

### ISSUE THREAD

**Issue by Mosilhy** [SISDevelop/SwarmPackagePy#6]
how to input the dataset  and use the WSA to it !

**Comment by khushbukella** [comment]
Hey MOsilhy, have you run this code?

**Comment by Mosilhy** [comment]
yes , i am trying to use SI to select features to send them to the classifier ( to avoid sending all the features)

**Comment by Anjalizi** [comment]
Has anyone been able to figure out the usage of this package?

**Comment by Mosilhy** [comment]
Totally useless package
